lib/client/api.go

```diff
@@ -2747,6 +2749,28 @@ func (tc *TeleportClient) runCommandOnNodes(ctx context.Context, clt *ClusterCli
 
 	g, gctx := errgroup.WithContext(ctx)
 	g.SetLimit(commandLimit(ctx, clt.AuthClient, mfaRequiredCheck.Required))
+
+	// Get the width of the terminal so we can wrap properly.
+	var width int
+	if stdoutFile, ok := tc.Stdout.(*os.File); ok {
+		conn, err := stdoutFile.SyscallConn()
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		var ctrlErr error
+		err = conn.Control(func(fd uintptr) {
+			width, _, ctrlErr = term.GetSize(int(fd))
+		})
+		if err != nil {
+			return trace.Wrap(err)
+		} else if ctrlErr != nil {
+			return trace.Wrap(ctrlErr)
+		}
```

Comment:
Swap the two variables?
```suggestion
		ctrlErr := conn.Control(func(fd uintptr) {
			width, _, err = term.GetSize(int(fd))
		})
		if ctrlErr != nil {
			return trace.Wrap(ctrlErr)
		} else if err != nil {
			return trace.Wrap(err)
		}
```